src/component/Header/Nav/Nav.js

```diff
@@ -13,9 +13,9 @@ export default ({ number, accountName, requestSignOut }) => {
         </NavLink>
         <Card accountName={accountName} requestSignOut={requestSignOut} />
       </div>
-      <NavLink to="/generation">
+      <a href="/generation">        
```

Comment:
why change this to <a>, it will reload the page?

Reply:
The NavLink was not loading the content of generation after corgi creation but that was due to the state of the contract.js not being updated; changed it and reverted this back to NavLink